admin_manual/groupware/mail.rst

```diff
@@ -93,6 +93,8 @@ LLM Processing
 
 The Mail app can optionally use large language models to process emails and offer assistance features like thread summaries, smart replies and event agendas.
 
+.. note:: The supported languages depend on the used large language model. Our prompts are written in english
```

Comment:
```suggestion
.. note:: The supported languages depend on the used large language model. Our prompts are written in English.
```

Not sure if we even need the second sentence. In my testing I had some German emails that worked fine. Just because the prompt is English might not necessarily mean that the data has to be English too.

Reply:
Idk if it's useful information either, I tested with french and arabic and it works.